src/seedu/addressbook/data/person/Person.java

```diff
@@ -83,5 +83,19 @@ public int hashCode() {
     public String toString() {
         return getAsTextShowAll();
     }
+    
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    public String getPrintableString(Printable... printables){
```

Comment:
Correct implementation of the varags method. As discussed in tutorial, there are many possible placement for this method.